docs/source/index.mdx

```diff
@@ -4,13 +4,14 @@ title: Introduction to Apollo Server
 
 **Apollo Server is an [open-source](https://github.com/apollographql/apollo-server), spec-compliant GraphQL server** that's compatible with any GraphQL client, including [Apollo Client](https://www.apollographql.com/docs/react). It's the best way to build a production-ready, self-documenting GraphQL API that can use data from any source.
 
-![diagram](./images/index-diagram.svg)
+![diagram](./images/frontend_backend_diagram.svg)
```

Comment:
```suggestion
<img src="./images/frontend_backend_diagram.svg" width="500" alt="Diagram showing Apollo Server bridging frontend and backend" />
```

Reply:
(The diagram is a little too big by default, and we can more easily shrink it a bit with raw HTML like this)

Reply:
Looks great! Thank you so much!